<!DOCTYPE HTML>
<!--
	Theory by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
--><html>
	<head>
		<title>Research</title>
		<meta charset="utf-8">
		<meta name="robots" content="index, follow, max-image-preview:large, max-snippet:-1, max-video-preview:-1">
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<link rel="stylesheet" href="assets/css/main.css">
	</head>
	<body class="subpage">

		<!-- Header -->
			<header id="header">
				<div class="inner">
					<a href="index.html" class="logo">Jongmin Kim Lab @ Cornell University</a>
					<nav id="nav">
						<a href="index.html">Home</a>
						<a href="people.html">People</a>
						<a href="research.html">Research</a>
						<a href="publications.html">Publications</a>
						<a href="news.html">News</a>
						<a href="photo.html">Photos</a>
						<a href="resource.html">Resources</a>
						<a href="recruitment.html">Join the lab</a>
						<a href="contact.html">Contact</a>
					</nav>
					<a href="#navPanel" class="navPanelToggle"><span class="fa fa-bars"></span></a>
				</div>
			</header>

		<!-- Main -->
			<section id="main" class="wrapper">
				<div class="inner">
					<header class="align-center">
						<h2>Research</h2>
						<!--JK: skip this line <p>Aliquam erat volutpat nam dui </p>-->
					</header>

				<!-- Content -->
					<h3 id="content">Gene Silencing</h3>
					<!--<div class="image fit">-->
						<center><img src="images/Re_SubsetON.gif" width="60%"></center>
					<!--</div>-->
					There are numerous genes in the genome, but a cell expresses only a subset of genes to establish its own identity. Thus, cell type-specific gene expression is at the core of cell fate determination. Gene silencing mechanisms play an essential role in transcriptional fidelity by blocking the misexpression of inappropriate genes. Our group is interested in the role of chromatin modifying proteins in preventing spurious transcription. In particular, <br><br>
					<ul><ul>
					<li> How is chromatin modified and organized to maintain the 'silent' state?  </li>
  					<li> How do the gene silencing mechanisms ensure cell type-specific gene expression and safeguard cell fate? </li>
  					<li> How do these mechanisms orchestrate the burst of testis-specific gene expression, support male fertility, and contribute to ensuing embryonic development? </li>
  					</ul></ul>
  					<br>

  					<h3 id="content">Spermatogenesis</h3>
					
					<center><img src="images/Re_Spermatogenesis.png" width="60%"></center>
					The testis has the most diverse transcriptome. Some genes have second testis-specific promoters that are regulated by testis-specific paralogous regulatory proteins. These new regulatory inventions drive dramatic gene expression changes. We will use this unique feature to observe the divergent behavior of chromatin regulators and advance our understanding of their mechanism. The male germ line will be the uniting principal model for our lab, and we will employ both fly and mouse models to utilize the unique advantages of each system. <br><br><br>
		


					<h3 id="content">Polycomb and CHD4/Zinc finger protein</h3>
					
					<center><img src="images/Re_PolycombCHD4.png" width="40%"></center>
					We will investigate the role of CHD4-Zinc finger protein (ZFP) and Polycomb complexes in male germ cells. Both complexes are required to prevent gene misexpression and can modulate chromatin architecture. However, they bind different sets of either active (CHD4-ZFP) and inactive (Polycomb) genes, suggesting that cells may have distinct mechanisms of compartmentalizing inactive and active chromatin. Furthermore, these proteins form distinct complexes in the testis by switching homologous components or by incorporating new subunits. We are interested in how changes in the subunits can alter the function of these complexes to support the unique chromatin environment of male germ cells. Ultimately, findings from these two chromatin complexes will lead to general principles on the segregation of active and inactive chromatin domains and their role in cell fate regulation.  <br><br><br>
		


					<h3 id="content">Model system: flies and mice</h3>
					
					<center><img src="images/Re_mice_and_fly.png" width="60%"></center>
					Why use two different systems? Several questions can be approached by studying more than one species. How did gene regulation evolve differently between flies and mice? What additional layers of regulation are needed in a mammalian genome that is 20 times the size of the fly genome? On the other hand, many genes involved in spermatogenesis are conserved in flies and mice. Then, what are the core processes required for the spermatogenesis? Flies will be a foundation for uncovering core mechanisms, and mice will provide the opportunity to address the complexity of distinct variant complexes that arose in mammals. Being in the Vet school will provide an opportunity to examine some of the many species between arthropods and mammals in the evolutionary tree. <br><br><br>

					<h3 id="content">Approach</h3>
					
					<center><img src="images/Re_approach.png" width="90%"></center>
					We employ diverse methods to address the research questions listed above. Our approach includes genetic perturbations, microscopy, and high-throughput sequencing. We identify new genes through genetic screens, exploring the consequences of perturbing the gene products using rapid induction and depletion methods. Cellular phenotypes are analyzed using classical histological techniques, Fluorescence In Situ Hybridization (FISH), and single-cell sequencing-based profiling. Gene expression and chromatin status are probed through sequencing-based chromatin profiling methods, such as Cut&Run and Hi-C. Additionally, we utilize FISH-based methods to analyze gene expression and chromatin structure in situ. We are also committed to developing innovative genetic screening methods and adapting existing chromatin profiling technologies for rare cell types within tissues. We strive to perform mechanistic studies in vivo through these complementing methodologies.<br><br>

					<!--The genome has numerous sites to initiate transcription, but a cell uses only a subset of those sites. We are interested in how chromatin structure and organization are modified to prevent spurious transcription. In particular, we will investigate how a burst of testis-specific transcription in spermatocytes is tightly regulated by Polycomb and CHD4-Zinc finger protein complexes. Our lab will explore the evolutionarily conserved repressive mechanisms safeguarding germ cell fate using flies and mice as model systems.-->
					<br><br>


					</p>

		<!-- Footer -->
			<footer id="footer">
				<div class="inner">
					<div class="flex">
						<ul class="icons">
							<li><a href="https://scholar.google.com/citations?user=sDNrClcAAAAJ&hl=en" class="icon fa-google"><span class="label">Google Scholar</span></a></li>
							<li><a href="https://twitter.com/JongminKmg" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
							<li><a href="https://github.com/jongminkmg" class="icon fa-github"><span class="label">Github</span></a></li>
							<li><a href="https://www.vet.cornell.edu/departments/biomedical-sciences" class="icon fa-university"><span class="label">BMS</span></a></li>
						</ul>
					</div>
				</div>
			</footer>
		<div class="copyright">
			Contact: <a href = "mailto: jk2938@cornell.edu">Jongmin Kim</a>
		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
